backend/wayland/backend.c

```diff
@@ -185,6 +186,28 @@ static const struct wl_drm_listener legacy_drm_listener = {
 	.capabilities = legacy_drm_handle_capabilities,
 };
 
+static uint32_t convert_wl_shm_format_to_drm(enum wl_shm_format format) {
```

Comment:
Note: when https://github.com/swaywm/wlroots/pull/2744 lands, this can be replaced with the centralized helper.